tests/unit/kats/generate_pq_hybrid_tls13_handshake_kats.py

```diff
@@ -0,0 +1,236 @@
+        "ec_shared_secret": "9348e27655539e08fffe46b35f863dd634e7437cc6bc11c7d329ef5484ec3b60",
+        "pq_shared_secret": "0A6925676F24B22C286F4C81A4224CEC506C9B257D480E02E3B49F44CAA3237F",
+        "transcript_hash": "f5f7f7867668be4b792159d4d194a03ec5cfa238b6409b5ca2ddccfddcc92a2b",
+    },
+    {
+        "group_name": "secp256r1_kyber512r3",
+        "cipher_suite": "TLS_AES_256_GCM_SHA384",
+        "ec_shared_secret": "9348e27655539e08fffe46b35f863dd634e7437cc6bc11c7d329ef5484ec3b60",
+        "pq_shared_secret": "0A6925676F24B22C286F4C81A4224CEC506C9B257D480E02E3B49F44CAA3237F",
+        "transcript_hash": "35412cebcf35cb8a7af8f78278a486fc798f8702eaebd067c97acb27bffe13524d8426a4ed57956b4fd0ffdc4c90be52",
+    },
+    {
+        "group_name": "x25519_bikel1r3",
+        "cipher_suite": "TLS_AES_128_GCM_SHA256",
+        "ec_shared_secret": "519be87fa0599077e5673d6f2d910aa150d7fef783c5e1491961fdf63b255910",
+        "pq_shared_secret": "1A88B3A458EE42906A5FD423817E043532579C4F79518A81213DC91D0F2FCEA9",
+        "transcript_hash": "f5f7f7867668be4b792159d4d194a03ec5cfa238b6409b5ca2ddccfddcc92a2b",
+    },
+    {
+        "group_name": "x25519_bikel1r3",
+        "cipher_suite": "TLS_AES_256_GCM_SHA384",
+        "ec_shared_secret": "519be87fa0599077e5673d6f2d910aa150d7fef783c5e1491961fdf63b255910",
+        "pq_shared_secret": "1A88B3A458EE42906A5FD423817E043532579C4F79518A81213DC91D0F2FCEA9",
+        "transcript_hash": "35412cebcf35cb8a7af8f78278a486fc798f8702eaebd067c97acb27bffe13524d8426a4ed57956b4fd0ffdc4c90be52",
+    },
+    {
+        "group_name": "secp256r1_bikel1r3",
+        "cipher_suite": "TLS_AES_128_GCM_SHA256",
+        "ec_shared_secret": "9348e27655539e08fffe46b35f863dd634e7437cc6bc11c7d329ef5484ec3b60",
+        "pq_shared_secret": "1A88B3A458EE42906A5FD423817E043532579C4F79518A81213DC91D0F2FCEA9",
+        "transcript_hash": "f5f7f7867668be4b792159d4d194a03ec5cfa238b6409b5ca2ddccfddcc92a2b",
+    },
+    {
+        "group_name": "secp256r1_bikel1r3",
+        "cipher_suite": "TLS_AES_256_GCM_SHA384",
+        "ec_shared_secret": "9348e27655539e08fffe46b35f863dd634e7437cc6bc11c7d329ef5484ec3b60",
+        "pq_shared_secret": "1A88B3A458EE42906A5FD423817E043532579C4F79518A81213DC91D0F2FCEA9",
+        "transcript_hash": "35412cebcf35cb8a7af8f78278a486fc798f8702eaebd067c97acb27bffe13524d8426a4ed57956b4fd0ffdc4c90be52",
+    },
+]
+
+def hdkf_extract(salt: bytes, key: bytes, hash_alg: str):
+    return hmac.new(salt, key, hash_alg).digest()
+
+def hkdf_expand_label(key: bytes, label: str, context: bytes, hash_alg: str):
+    label_arr = [0, hashlib.new(hash_alg).digest_size, len("tls13 ") + len(label)]
+    for c in "tls13 ":
+        label_arr.append(ord(c))
+
+    for c in label:
+        label_arr.append(ord(c))
+
+    label_arr.append(len(context))
+
+    for c in context:
+        label_arr.append(c)
+
+    return hdkf_extract(key, bytearray(label_arr) + bytes([1]), hash_alg)
```

Comment:
why the `+ bytes([1])`?

Reply:
This is concatenating a single byte with value of `0x01` to the end label array, I believe this is part of the HKDF standard. See: https://datatracker.ietf.org/doc/html/rfc5869#section-2.3

Page 4 of the RFC also concatenates the byte `0x01`:
```
T(1) = HMAC-Hash(PRK, T(0) | info | 0x01)
```

As does the Wikipedia Python implementation: 
https://en.wikipedia.org/wiki/HKDF#Example:_Python_implementation

Reply:
thanks!